Build failure with PostgreSQL 10.x

tsearch_extras 0.3 fails to build with pgsql 10.3 with this error:

```c
gmake[1]: Entering directory '/wrkdirs/usr/ports/databases/tsearch_extras/work/tsearch_extras-0.3'
cc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -Wno-unused-command-line-argument -O2 -pipe  -fstack-protector -fno-strict-aliasing -fPIC -DPIC -I. -I./ -I/usr/local/include/postgresql/server -I/usr/local/include/postgresql/internal  -I/usr/local/include -I/usr/local/include -I/usr/local/include  -c -o tsearch_extras.o tsearch_extras.c
tsearch_extras.c:82:47: error: too few arguments to function call, expected 3, have 2
                                                                                          (Node *) makeString(pstrdup("1"))));
                                                                                                                           ^
/usr/local/include/postgresql/server/nodes/makefuncs.h:83:1: note: 'makeDefElem' declared here
extern DefElem *makeDefElem(char *name, Node *arg, int location);
^
1 error generated.
```

Looks like definition of `makeDefElem()` has changed to require additional third `location` argument. 

```c
DefElem *
makeDefElem(char *name, Node *arg, int location)
```

Can this be fixed conditionally for pgsql10?

--- a/tsearch_extras.c
+++ b/tsearch_extras.c
@@ -77,9 +77,15 @@ ts_match_locs_setup(Oid cfgId, TsMatchesData *mdata, text* in, TSQuery query)
 
 	hlparsetext(cfgId, &prs, query, VARDATA(in), VARSIZE(in) - VARHDRSZ);
 
+	#if PG_VERSION_NUM >= 100000
+	headline_options = lappend(headline_options,
+											  makeDefElem(pstrdup("HighlightAll"),
+											  (Node *) makeString(pstrdup("1")), -1));
+	#else
 	headline_options = lappend(headline_options,
 											  makeDefElem(pstrdup("HighlightAll"),
 											  (Node *) makeString(pstrdup("1"))));
+	#endif
 
 	FunctionCall3(&(prsobj->prsheadline),
 				  PointerGetDatum(&prs),

